dashboard-app/src/lib/api/index.ts

```diff
@@ -1,10 +1,12 @@
 import _axios, { AxiosRequestConfig, Method } from 'axios';
-import { Api } from '../../../../api/src/api/v1/types/api';
+import { Api } from '../../../../api/src/api/v1.1/types/api';
 import qs from 'qs';
 
+const API_VERSION = 'v1.1';
+
 const baseURL = process.env.REACT_APP_API_HOST_DOMAIN ?
-  `${process.env.REACT_APP_API_HOST_DOMAIN}/v1`
-  : '/v1';
+  `${process.env.REACT_APP_API_HOST_DOMAIN}/${API_VERSION}`
+  : `/${API_VERSION}`;
```

Comment:
Out of curiosity, why did you choose to store the api version as an environment variable?

Reply:
Is the question why _did_ I or why _didn't_ I? Cos it's not currently an env var. It could be. Maybe it should. But I guess the env var only matters at build time on `staging` (because we don't rebuild the slug when we promote). Also the host changes between local/prod environments, but the API version doesn't.

Reply:
Oh I missed the line this variable was declared. I just assumed it was an env var because of the casing

Reply:
On second view, I may be just be giving an unrequested review. My bad.